composer.json

```diff
@@ -42,5 +42,6 @@
     },
     "config": {
         "sort-packages": true
-    }
+    },
+    "scripts": { "test": [ "phpunit" ] }
```

Comment:
:octocat: < Please format this into multi-lines

Reply:
i did it 👍